aksel.nav.no/website/pages/templates/soknad-introside-aap/soknad-introside-aap.tsx

```diff
@@ -0,0 +1,314 @@
+import { useEffect } from "react";
+import { ArrowRightIcon } from "@navikt/aksel-icons";
+import {
+  Accordion,
+  Bleed,
+  BodyLong,
+  BodyShort,
+  Box,
+  Button,
+  Checkbox,
+  GuidePanel,
+  Heading,
+  Link,
+  List,
+  Page,
+  Show,
+  Stack,
+  VStack,
+} from "@navikt/ds-react";
+
+function Example() {
+  useDekorator();
+
+  return (
+    <Page data-aksel-template="form-intropage-v1" footer={<Footer />}>
```

Comment:
Consider moving `data-aksel-template` a bit down in the DOM since some devs probably will not copy this part (b.c. they already have implemented header, footer etc. another place in the code).

(See existing templates)